File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/ExpedienteRecomendacionGeneralFormDTO.cs
namespace TrackrAPI.Dtos.GestionExpediente;
public class ExpedienteRecomendacionGeneralFormDTO
{
    public int? IdExpedienteRecomendacionesGenerales {  get; set; }
    public int Tipo { get; set; }
    public string? Descripcion { get; set; }
    public int IdDoctor { get; set; }
    public DateTime Fecha { get; set; }
    public int? IdPadecimiento { get; set; }
    public List<int>? Paciente { get; set;}
}

File: Frontend/trackr-administrador/src/app/shared/dtos/gestion-expediente/expediente-recomendacion-general/expediente-recomendacion-general-form-dto.ts
export class ExpedienteRecomendacionGeneralFormDTO{
    public idExpedienteRecomendacionGeneral ?: number;
    public tipo: number;
    public descripcion : string;
    public idDoctor ?: number;
    public fecha : Date;
    public idPadecimiento ?: number;
    public paciente ?: number[];
}